drawings-gods_data-lists.ttl

```diff
@@ -4994,9 +4994,9 @@
 				knora-base:listNodeName "nakano" ;
 				knora-base:listNodePosition 2 ;
 				rdfs:label "Nakano"@fr , "Nakano"@en .
-
+				
 			<http://rdfh.ch/lists/0105/drawings-gods-2016-list-VillageHList-fuchu> a knora-base:ListNode ;
-				knora-base:hasRootNode <http://rdfh.ch/lists/0105/drawings-gods-2016-list-VillageHList> ;
+				knora-base:hasRootNode <http://rdfh.ch/lists/0105/drawings-gods-2016-list-SchoolHList> ;
```

Comment:
`knora-base:hasRootNode` should be `<http://rdfh.ch/lists/0105/drawings-gods-2016-list-VillageHList>` instead of `<http://rdfh.ch/lists/0105/drawings-gods-2016-list-SchoolHList>` I think.